.dockerignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.git
+website
```

Comment:
The .dockerignore file manages the build context that gets sent to the service during the build process. In Habitica's case, we're building a stand-alone image that pulls its own`git clone` of Habitica, builds it and serves it, so there is little that _must_ be passed to the docker service (theoretically, everything except Dockerfile could be excluded). The /website directory gets created within the image as part of its local Habitica repo and build process, so it doesn't need to be passed on to docker.